### ISSUE TITLE
Error with detached element

### ISSUE THREAD

**Issue by louisameline** [eddiemf/vue-scrollactive#44]
Hi,

Thanks for the plugin!

An error I encountered: I use scrollActive on an element which is in a tab. When I switch tabs and scroll, I get the following error:

```
vue-scrollactive.js?ce7a:1088 Uncaught TypeError: Cannot read property 'offsetHeight' of null
    at eval (vue-scrollactive.js?ce7a:1088)
```

which is this line:

```
var isScreenBeforeSectionEnd = window.pageYOffset < _this.getOffsetTop(target) - _this.offset + target.offsetHeight;
```

I guess the inactive tab is detached by the framework I use (Quasar) but not destroyed (because I use the keep-alive property of tabs, which keeps the rendered HTML in memory), so scrollActive is still listening to events and doing things. Not sure if you can fix it without me worrying about it, or if I should call some method to put it on hold while the tab is inactive?
Thank you.

**Comment by eddiemf** [comment]
Hello, are you sure you're using the latest version of the plugin? Because the line you're referring to contains this check right above:

```
if (!target) return;
const distanceFromTop = this.scrollContainer.scrollTop || window.pageYOffset;
const isScreenPastSection = distanceFromTop >= this.getOffsetTop(target) - this.offset;
const isScreenBeforeSectionEnd = distanceFromTop
  < (this.getOffsetTop(target) - this.offset) + target.offsetHeight;
``` 

So it's impossible for `target` to be `null` since it would just return inside the condition. Also the code you pasted is slightly different since it's checking `window.pageYOffset` instead of `this.scrollContainer.scrollTop` (which was implemented in one of the latest updates).

Try to update the plugin and check it again, I'm pretty sure it should solve the issue.

**Comment by louisameline** [comment]
Right, I was still in 0.6, fixed in 0.8, thank you!
